tests/ast/test_ast_driver.py

```diff
@@ -28,6 +31,7 @@ class TestCase:
     source: str
     expected_ast_base64: str
     expected_ast_unparsed: str
+    __test__: bool = False  # Add this to suppress pytest collection warning
```

Comment:
Set this to `True` ...; also period at the end.

Reply:
It's purposefully set to False, this is because pytest will assume any class starts with `Test` is a testcase (https://docs.pytest.org/en/7.1.x/explanation/goodpractices.html#conventions-for-python-test-discovery). This behavior can be suppressed with `__test__:bool = False` which will get rid off a warning that this class does not represent a valid test case (in the pytest way).